hedera-node/src/test/java/com/hedera/services/txns/file/FileDeleteTransitionLogicTest.java

```diff
@@ -9,9 +9,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I think we should keep the `verify` methods used in the unit tests. ( where applicable, you can check  #536  for reference)